src/util/feed.tsx

```diff
@@ -0,0 +1,28 @@
+import { metadataService } from '@zer0-os/zos-zns';
+
+export const fetchMetadata = async (metadataUrl, metadataService, abortController) => {
+  if (!metadataUrl) return;
+
+  try {
+    // Consider moving this into MetadataService. The MetadataService is based on SuperAgent which doesn't support the AbortController
+    const response = await fetch(metadataUrl, { signal: abortController.signal });
+    return metadataService.normalize(await response.json());
+  } catch (_e) {
+  }
+}
+
+export const augmentWithMetadata = async (item, metadataService, abortController) => {
```

Comment:
i think this is the wrong level for interacting with metadata. metadata is an underlying implementation detail of a system we're using. we've encapsulated that knowledge in the zos-zns library so that at this level we can just ask for the domain object that we're concerned with.

let's pull this out into the zns client, or at least as far as the saga for now.